demos/smart_classroom_demo/README.md

```diff
@@ -121,11 +121,11 @@ Example of a valid command line to run the application for recognizing first rai
 The demo uses OpenCV to display the resulting frame with labeled actions and faces.
 
 > **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo has been tested on the following Model Downloader available topologies: 
-* `face-detection-adas-0001`
-* `landmarks-regression-retail-0009`
-* `face-reidentification-retail-0095`
-* `person-detection-action-recognition-0005`
-All other models may work incorrectly on these devices.
+>* `face-detection-adas-0001`
+>* `landmarks-regression-retail-0009`
+>* `face-reidentification-retail-0095`
+>* `person-detection-action-recognition-0005`
```

Comment:
I think @IRDonch's comment on alphabetical order should be applied to all such added lists.

Reply:
Done